README.md

```diff
@@ -100,6 +100,13 @@ path = "M/60'/0'/0'"
 xpub = BlockKeys.CKD.derive(root_key, path)
 ```
 
+### Solana
+
+```
+path = "M/501'/0'/0'"
+xpub = BlockKeys.CKD.derive(root_key, path, network: :solana)
```

Comment:
There are a few other networks that use the same `ed25519` curve, would it make more sense to call the parameter something like `curve` ? That way it would make it more clear why Ethereum and Bitcoin don't need their own `network` options, plus it will help if we potentially want to add address derivations for those ?

Reply:
Thanks for your great suggestion @tzumby. The keyword has been updated to `curve: :ed25519`.